airship/AIR-138_kubeadm-rotating-ca.md

```diff
@@ -41,7 +41,42 @@ In case of HA cluster, the certificate renewal has to be done separately on each
 
 The validity of the certificates can be set while generating CSRs. 
 
-In Kubeadm there are options to renew/rotate certificate automatically and manually. Kubadm does automatic renewal of all certificates during control plane upgrade (We haven’t checked if it works as expected since we are more focused on manual certificate renewal). The manual certificate renewal/rotation is performed as described in previous comments. The ```renew``` command in alpha phase allows to renew the following certificates
+In Kubeadm there are options to renew/rotate certificate automatically and manually. 
+
+### Automatic Certificate Renewal 
+Kubadm does automatic renewal of all certificates during control plane upgrade. Our experiment findings with this use case is as follows:
+1. Test case: Upgrading cluster from version: 1.14.0 to 1.15.0
+2. During the upgrader process kubeadm renews the following certificates:
```

Comment:
2 here seems a result ?. If so remove it and add indentation

Reply:
Done